server/db/models/Order.js

```diff
@@ -0,0 +1,18 @@
+const Sequelize = require('sequelize');
+const { UUID, UUIDV4, BOOLEAN } = Sequelize;
+const db = require('../database');
+
+const Order = db.define('order', {
+  id: {
+    primaryKey: true,
+    type: UUID,
+    defaultValue: UUIDV4
+  },
+  complete: {
+    type: BOOLEAN,
+    allowNull: false,
+    defaultValue: false
+  }
```

Comment:
If you have the other virtual field, you can have a virtual field here, that adds all those up to create the total price on the fly.